azure-pipelines.yml

```diff
@@ -403,14 +403,14 @@ stages:
             MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\MSBuild.exe"
             PLATFORMTOOLSET: "v140"
             WINDOWSSDKTARGET: "10.0.17763.0"
-          vs2019_dev:
-            SQ_VERSION: "DEV"
-            SONAR_CFAMILYPLUGIN_VERSION: "LATEST_RELEASE"
-            SONAR_CSHARPPLUGIN_VERSION: "LATEST_RELEASE"
-            SONAR_VBNETPLUGIN_VERSION: "LATEST_RELEASE"
-            MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\MSBuild\\Current\\Bin\\MSBuild.exe"
-            PLATFORMTOOLSET: "v140"
-            WINDOWSSDKTARGET: "10.0.17763.0"
+          # vs2019_dev:
```

Comment:
could you also comment out `vs2017_dev` and change `SQ_VERSION` to DEV? it seems there's a bug there because we run VS 2017 tests with LATEST_RELEASE twice...

Reply:
In this PR I do the renaming of In PR #1179 I will do the renaming of vs2017_dev to vs2017_latest89 and target the latest LTS version. The only question is do we also want a  VS2017 one which really targets dev?

Reply:
If yes I'll change that PR and I would do the commenting there.

Reply:
in any case, as this was unblokcing master, indeed it should have been a different PR